.github/workflows/actions.yml

```diff
@@ -0,0 +1,82 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-linux:
+    name: Linux Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up GraalVM
+      uses: ayltai/setup-graalvm@v1
+      with:
+        java-version: 11
+        graalvm-version: 20.3.0
+        native-image: true
+    - name: Build
+      env:
+        ACCESS_KEY: ${{ secrets.ACCESS_KEY }}
+      run: mvn clean verify -Pfailsafe
+    - name: 'Upload Artifact'
+      if: ${{ always() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: target-linux
+        path: ./target
+        retention-days: 5
+
+  build-windows:
+    name: Windows Build
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up GraalVM
+      uses: ayltai/setup-graalvm@v1
+      with:
+        java-version: 11
+        graalvm-version: 20.3.0
+        native-image: true
+    - name: Install VS tools
+      run: choco install visualstudio2017-workload-vctools
+    - name: Build
+      env:
+        ACCESS_KEY: ${{ secrets.ACCESS_KEY }}
+      run: ./build-windows.bat
+    - name: 'Upload Artifact'
```

Comment:
reuse with YAML anchor from build-linux? (e.g. `- *upload-artifact`), might require setting a ENV var for `name` or maybe retrieve it from an already defined ENV var, e.g. CI_JOB (dunno if it exists)?

Reply:
See comment above